javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+let count = 4;
+function updateHeader() {
+  $countdownDisplay.textContent = count;
+
+  if (count === 0) {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(intervalID);
+  }
+  count--;
```

Comment:
`count` should decrement before line 7. Otherwise with your current `1500` miliseconds it would take three seconds to see the first decrement.